Insights/Account/UEBAAnomaly.yaml

```diff
@@ -0,0 +1,88 @@
+      - ProjectedName: Uncommon in Tenant
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.CountryUncommonlyConnectedFromInTenant == True)"
+      - ProjectedName: First Time Tenant Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTimeConnectionFromCountryObservedInTenant == True)"
+    - Filter: " where (ActionType in('InteractiveLogon','Sign-in','ResourceAccess') and (ActivityInsights.FirstTimeDeviceObservedInTenant == True or ActivityInsights.FirstTImeUserConnectedFromDevice == True or ActivityInsights.DeviceUncommonlyUsedAmongPeers == True or ActivityInsights.FirstTimeResourceAccessedInTenant == True or ActivityInsights.ResourceUncommonlyAccessedAmongPeers == True or ActivityInsights.FirstTimeUserAccessedResource == True))"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTImeUserConnectedFromDevice == True or Evidence.FirstTimeUserAccessedResource == True),['Uncommon Among Peers']=countif(Evidence.DeviceUncommonlyUsedAmongPeers == True or Evidence.ResourceUncommonlyAccessedAmongPeers == True), ['Uncommon in Tenant']=0 , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeDeviceObservedInTenant == True or Evidence.FirstTimeResourceAccessedInTenant == True)
+      Project: project Anomaly = 'Anoamlaous access', ['First Time User Anomaly'], ['Uncommon Among Peers'], ['Uncommon in Tenant'], ['First Time Tenant Anomaly']
+      LinkColumnsDefinitions:
+      - ProjectedName: First Time User Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTImeUserConnectedFromDevice == True or Evidence.FirstTimeUserConnectedToResource == True)"
+      - ProjectedName: Uncommon Among Peers
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.PeersConnectedToDestinationDevice == True or Evidence.PeersConnectedToResource == True)| where UserPrincipalName == '{{Account_UPN}}'"
+      - ProjectedName: Uncommon in Tenant
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.DestinationDeviceUncommonFromInTenant == True)"
+      - ProjectedName: First Time Tenant Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTImeTenantConnectedFromDevice == True or Evidence.FirstTimeTenantConnectedToResource == True)"
+    - Filter: " where (ActionType in('Create role assignment','Export an existing database','Remove database vulnerability assessment rule baseline','Run Command on Virtual Machine') and (ActivityInsights.FirstTimeUserPerformedAction == True or ActivityInsights.FirstTimeActionPerformedInTenant == True or ActivityInsights.ActionUncommonlyPerformedAmongPeers == True) )"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTimeUserPerformedAction == True),['Uncommon Among Peers']=countif(Evidence.ActionUncommonlyPerformedAmongPeers == True), ['Uncommon in Tenant']=countif(Evidence.ActionUncommonlyPerformedFromInTenant==True) , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeActionPerformedInTenant == True)
```

Comment:
Need to add 'by UPN' at the end, otherwise when we don't have a match, we will get rows with 0 returns which is not how we want no results to show up in the UX.